src/models/converters/transactions/tests/summary.rs

```diff
@@ -152,6 +166,61 @@ async fn module_tx_to_summary_transaction_failed() {
     assert_eq!(actual, expected);
 }
 
+#[rocket::async_test]
+async fn module_transaction_to_custom_summary_and_module_info() {
+    let module_tx = serde_json::from_str::<ModuleTransaction>(crate::json::MODULE_TX).unwrap();
+
+    let mut mock_info_provider = MockInfoProvider::new();
+    mock_info_provider.expect_safe_info().times(0);
+    mock_info_provider.expect_token_info().times(0);
+    mock_info_provider
+        .expect_address_ex_from_contracts()
+        .times(1)
+        .return_once(move |address| {
+            Ok(AddressEx {
+                value: address.to_string(),
+                name: Some(format!("{}_name", address)),
+                logo_url: None,
```

Comment:
This should be adjusted after you rebase. Some how the PR is still mergable :/

Reply:
Yeah the checks were done on a state where the code that was added didn't have any conflicts (currently it still doesn't have conflicts so we should rebase).